src/shared/ui/list/list.tsx

```diff
@@ -0,0 +1,44 @@
+// todo delete TextWithLink
+import classNames from 'classnames/bind';
+import { TextWithLink } from '../text-with-link';
+import { LinkList } from '@/widgets/required/required.types';
+
+import styles from './list.module.scss';
+
+export type ListProps = {
+  data: ListData;
+  marked?: boolean;
+};
+
+export type ListData = (string | LinkList)[] | [] | undefined;
+
+export const cx = classNames.bind(styles);
+
+export const List = ({ data, marked = true }: ListProps) => {
```

Comment:
But we've unmarked lists in some places. `List` - it's `<ul>` tag - 'unordered list', so I don't think there's any need to add anything to its' name.

Reply:
Ok, another way - will this component replace all lists in the codebase, ul, ol, implicit ones (if any)?

Reply:
`ul` and implicit - yes it is. `ol` - we haven't it now

Reply:
We have one, thats why I'm asking
https://github.com/rolling-scopes/site/blob/c9275f0a7adbc0978f45369c4c594565cb6ac5cc/src/widgets/faq/ui/faq.tsx#L16

Reply:
I don't think we should use `List` component here because this `<ol>` list has specific structure and this widget is not reusable - it is used only on one page. So I don't see any point in complicating the structure of the `List` component because of it.

Reply:
What I'm trying to say is, naming of component could paly crucial role, and then lead to bug or incorrect use.